### PR TITLE
TELCODOCS-1346-OCP: New UI field on the vSphere Configuration page - …

### PR DESCRIPTION
**Pull request by tshwartz**
Affected versions: 
main, 4.13, 4.12, 4.14

Summary of Change:

Added the vSphere Connection field to the overview and procedure, in the 'Configuring the vSphere connection settings after an installation' section of the OpenShift Container Platform document.

Related JIRA:
https://issues.redhat.com/browse/TELCODOCS-1346

Print Preview:
https://60886--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-post-installation-configuration.html

SME: Elior Erez
QE: Yuri Obshansky

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by tshwartz** [comment]
Ready for review. Question about OCP version.

**Review by eliorerz — APPROVED** [review]
/lgtm

File: modules/configuring-vsphere-connection-settings.adoc
Comment (skopacz1) [inline modules/configuring-vsphere-connection-settings.adoc:31]: Per the supplementary style guide's section on admonitions, this reads more like an [IMPORTANT](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions:~:text=performance%2C%20or%20supportability.-,IMPORTANT,-Advisory%20information%20essential) admonition to me.

However, it's tricky because I suppose this note might not be "essential" to users with 4.12 clusters.

File: modules/configuring-vsphere-connection-settings.adoc
Comment (skopacz1) [inline modules/configuring-vsphere-connection-settings.adoc:33]: This suggestion roughly follows the IBM style guide's examples for [References to earlier or later versions](https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions#references-to-earlier-or-later-versions)

```suggestion
This step is mandatory for {product-title} 4.13 or later.
```

**Review by skopacz1 — COMMENTED** [review]
I left some suggestions, but otherwise looks good! Just a few additional things:

1. Your comment zero says to apply these changes to main, 4.12, and 4.13. I think the `enterprise-4.14` branch is available now, so if this PR is supposed to be applied to 4.12+, please be sure to mention 4.14 as well.
2. I am new to peer review, so please wait until @adellape approves of my review before applying any feedback. Thank you!

**Review by adellape — COMMENTED** [review]
@skopacz1 @tshwartz Peer review looks good to me 👍

File: modules/configuring-vsphere-connection-settings.adoc
Comment (tshwartz) [reply modules/configuring-vsphere-connection-settings.adoc:31]: I'll change it to Important. They can't really make a mistake though, as the asterisk will be there from 4.13 onwards. It's more to tell users of earlier versions that they can skip that field if they don't have the cluster name.

Comment (tshwartz) [reply modules/configuring-vsphere-connection-settings.adoc:31]: I've also added 4.14 to the merge versions. Thanks for your feedback!

**Review by sheriff-rh — APPROVED** [review]
lgtm, merging.

**Pull request merged**
